### PR TITLE
Add ECE489

### PR DESCRIPTION
**Pull request by kyle11067**

### CODE REVIEW

File: docs/Course Wiki/ECE Course Offerings/ECE489.md [hunk 1]
```diff
@@ -0,0 +1,38 @@
...
+
+## Content Covered
+
+This class has no exams. Instead the content is partitioned into homework (50%), and labs/final project (50%). In lecture, the following sections were covered: kinematics of serial link robots, robot dynamics, independent joint control, position/motor control, and force/torque control. Topics were added in addition to the course text (Robot Modeling and Control, Spong, Hutchinson & Vidyasagar, Wiley) in the second half of the semester, which were detailed in PowerPoint notes posted on Compass2g. Lab exercises followed the topics covered in the lecture, written about kinematic transformations, equations of motion, PD/PID joint control, inverse dynamics joint control, and task space PD and impedance control, respective to their execution in lab. Each was implemented on a serial link manipulator, called the CRS robotic arm; predominantly, the labs entailed programming in C to obtain a desired output from the robot. The final project for the course was an obstacle course that was executed by the lab robot. It needed to place a peg inside of a slot, avoid an obstacle, follow a zig-zag channel, and lightly press against an object on a spring without breaking it.
```
Comment (V-001) [inline docs/Course Wiki/ECE Course Offerings/ECE489.md:14]: Add the name of the course here

File: docs/Course Wiki/ECE Course Offerings/ECE489.md [hunk 2]
```diff
@@ -0,0 +1,38 @@
+---
+tags:
+- Robotics
+- Control systems
+- Classical mechanics
+---
+
+# ECE489
+
+ECE489/ME446/SE422 is a 4-credit-hour course that satisfies the Technical Electives requirement for ECE majors and satisfies a hardware lab for EEs. It is offered only in spring semesters.
```
Comment (V-001) [inline docs/Course Wiki/ECE Course Offerings/ECE489.md]: add the name of the course here

**Review by V-001 — DISMISSED** [review]
address comments

**Review by lnguy7 — APPROVED** [review]
rerequest done LGTM

**Pull request merged**